service/src/test/java/bio/terra/externalcreds/services/OAuth2ServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package bio.terra.externalcreds.services;
```

Comment:
Could we move this to, like, `bio.terra.manualtests`? Just so that everything in `bio.terra.externalcreds` in the test folder is all jUnit tests, which is what I'd expect.

Reply:
I like the package structure of the test to mirror the package structure of the thing being tested. That's where I'd go looking for it.